package.json

```diff
@@ -275,6 +310,28 @@
           "command": "one.toolchain.uninstall",
           "when": "view == ToolchainView && viewItem =~ /toolchain/ && !one.job:running",
           "group": "inline"
+        },
+        {
+          "command": "one.viewer.metadata.show",
```

Comment:
Could you remove these `Metadata Viewer`-related lines? (4)

Reply:
ok, I`ll apply that part.